examples/map.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey"
+)
+
+// the questions to ask
+var simpleQs = []*survey.Question{
+	{
+		Name: "name",
+		Prompt: &survey.Input{
+			Message: "What is your name?",
+		},
+		Validate: survey.Required,
+	},
+	{
+		Name: "color",
+		Prompt: &survey.Select{
+			Message: "Choose a color:",
+			Options: []string{"red", "blue", "green"},
+		},
+		Validate: survey.Required,
+	},
+}
+
+func main() {
```

Comment:
I assume you're talking about using AskOne and passing in the type I want out. This definitely works, but if I have 5+ prompts with various types and need to eventually get them into a map[string]interface{} to pass them as data to a go template, that's a lot more code on the caller side then simply passing slice of questions to Ask and getting back the proper types.
Doing it one by one is a bit of a pain.